docs/components/secnav/test.html

```diff
@@ -7,7 +7,7 @@
 <div id="vue-secNavDemo">
```

Comment:
@danielmdesigns, please see the deploy preview full-page demo:

https://deploy-preview-785--helix-ui.netlify.app/components/secnav/test.html

Reply:
![secondary-nav-padding](https://user-images.githubusercontent.com/5085619/98729967-43217c80-2361-11eb-920a-9738f7b7973a.png)

Looks good except the top/bottom padding is 4px and should be 8px. Screenshot attached.